addon/components/list-keyboard-navigator.js

```diff
@@ -73,7 +73,9 @@ export default class ListKeyboardNavigatorComponent extends Component {
     if (_highlightedBy === 'mouse') return;
     if (highlightedIndex < 0) return;
 
-    let el = this.scrollElement.querySelector(`${itemSelector}:nth-of-type(${highlightedIndex + 1})`);
+    const el = this.scrollElement.querySelectorAll(itemSelector)[highlightedIndex];
+    if (!el) { return; }
+
```

Comment:
I happened to scroll down a bit and saw the same bit of code there.
https://github.com/cph/ember-list-keyboard-navigator/pull/17/files#diff-2e9e00f753d668ea16d4783550815180R99
Do we need to address that one as well?

Reply:
Good call, Luke! Yeah, that one will need to be fixed, as well. 👍